### PR TITLE
Fix trove

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by vmoravec** [comment]
+1
One note though: trove configuration in `ssl-insecure` batch files is still disabled due to https://bugs.launchpad.net/trove/+bug/1535895 , can we enable it or not yet?

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2407,6 +2407,9 @@ function custom_configuration()
                 proposal_set_value rabbitmq default "['attributes']['rabbitmq']['ha']['storage']['drbd']['size']" "$drbd_rabbitmq_size"
                 proposal_set_value rabbitmq default "['deployment']['rabbitmq']['elements']['rabbitmq-server']" "['cluster:$clusternamedata']"
             fi
+            if iscloudver 5plus; then
+                proposal_set_value rabbitmq default "['attributes']['trove']['enabled']" true
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: test failed with
-e:6:in `<main>': undefined method `[]=' for nil:NilClass (NoMethodError)
+ complain 30 'rubyjsonparse 
        j['\''attributes'\'']['\''trove'\'']['\''enabled'\'']=true
        puts JSON.pretty_generate(j)

Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: Should be `proposal_set_value rabbitmq default "['attributes']['rabbitmq']['trove']['enabled']" true`

**Comment by dirkmueller** [comment]
@vmoravec I don't know, I just wanted to keep the crowbar batch yamls in sync for now (except for the ssl differences)

**Pull request merged**